TopAppBar/TopAppBar.jsx

```diff
@@ -0,0 +1,94 @@
+import {h} from 'preact';
+import MaterialComponent from '../MaterialComponent';
+import {MDCTopAppBar} from '@material/top-app-bar';
+class TopAppBar extends MaterialComponent {
+  constructor() {
+    super();
+    this.componentName = 'top-app-bar';
+    this._mdcProps = ['short', 'short-collapsed'];
+    this._onNav = this._onNav.bind(this);
+  }
+  _onNav(e) {
+    if (this.props.onNav) {
+      this.props.onNav(e);
+    }
+  }
+  componentDidMount() {
+    this.MDComponent = new MDCTopAppBar(this.control);
+    this.MDComponent.listen('MDCTopAppBar:nav', this._onNav);
+  }
+  componentWillUnmount() {
+    this.MDComponent.unlisten('MDCTopAppBar:nav', this._onNav);
+    this.MDComponent.destroy && this.MDComponent.destroy();
```

Comment:
It says in the API documentation that `destroy()` is a property to the component (see [mdc-base](https://github.com/material-components/material-components-web/tree/master/packages/mdc-base)).

Reply:
Done